src/shared/proto/messages_robocup_ssl_detection_tracked.proto

```diff
@@ -0,0 +1,120 @@
+syntax = "proto2";
+
+// A vector with two dimensions
+message Vector2 {
+    required float x = 1;
+    required float y = 2;
+}
+
+// A vector with three dimensions
+message Vector3 {
+    required float x = 1;
+    required float y = 2;
+    required float z = 3;
+}
+
+// The team color of the robot
+enum TeamColor {
+    // team not set
+    TEAM_COLOR_UNKNOWN = 0;
+    // yellow team
+    TEAM_COLOR_YELLOW = 1;
+    // blue team
+    TEAM_COLOR_BLUE = 2;
+}
+
+// A unique robot id with team information
+message RobotId {
+    // The robot number
+    required uint32 id = 1;
+    // The team color
+    required TeamColor team_color = 2;
+}
+
+// Capabilities that a source implementation can have
+enum Capability {
+    CAPABILITY_UNKNOWN = 0;
+    CAPABILITY_DETECT_FLYING_BALLS = 1;
+    CAPABILITY_DETECT_MULTIPLE_BALLS = 2;
+    CAPABILITY_DETECT_KICKED_BALLS = 3;
+}
+
+// A single tracked ball
+message TrackedBall {
+    // The position (x, y, height) [m] in the ssl-vision coordinate system
+    required Vector3 pos = 1;
+
+    // The velocity [m/s] in the ssl-vision coordinate system
+    optional Vector3 vel = 2;
+
+    // The visibility of the ball
+    // A value between 0 (not visible) and 1 (visible)
+    // The exact implementation depends on the source software
+    optional float visibility = 3;
+}
+
+// A ball kicked by a robot, including predictions when the ball will come to a stop
+message KickedBall {
+    // The initial position [m] from which the ball was kicked
+    required Vector3 pos = 1;
```

Comment:
Is it safe to assume that balls are always kicked from the ground?

In this case, I would make this a Vector2